plugins/modules/user_role_workflow_manager.py

```diff
@@ -1239,7 +1241,7 @@ def get_diff_merged(self, config):
                     consolidated_data, update_required_param = self.role_requires_update(self.have["current_role_config"], desired_role)
 
                     if not consolidated_data:
-                        self.msg = "Role does not need any update"
+                        self.msg = "Role with role_name '{0}' already exist and the role does not need any update".format(self.have.get("role_name"))
```

Comment:
The word "exist" should be "exists" .. 

`self.msg = "Role with role_name '{0}' already exists and does not require an update.".format(self.have.get("role_name"))`

Reply:
Updated the code accordingly.